src/gcp/rules.ts

```diff
@@ -1,14 +1,14 @@
 "use strict";
```

Comment:
remove as 'use strict' is not necessary in TS files

Reply:
done